Source/VoidPromise.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 extension Promise where T == Void {
     
     public convenience init(callback: @escaping (
-        _ resolve: @escaping (() -> Void),
```

Comment:
These two seem to compile untouched on my side as well

Reply:
Not for me after upgrading an existing 4.2 project to swift 5 inside xcode 10.2.1

Reply:
Yeah, for me neither. I'll look into recloning the project and seeing if I can reproduce why it's building on @s4cha's machine.